sim/scripts/create_fixed_torso.py

```diff
@@ -27,9 +34,35 @@ def update_urdf() -> None:
                 upper = str(limits.get("upper", 0.0))
                 limit.set("lower", lower)
                 limit.set("upper", upper)
+
+                for key, value in effort.items():
+                    if key in joint_name:
+                        limit.set("effort", str(value))
+
+                for key, value in velocity.items():
+                    if key in joint_name:
+                        limit.set("velocity", str(value))
+
+            dynamics = joint.find("dynamics")
+            if dynamics is not None:
+                for key, value in friction.items():
+                    if key in joint_name:
+                        dynamics.set("friction", str(value))
+
     # Save the modified URDF to a new file
     tree.write("sim/stompy_legs/robot_fixed.urdf")
 
 
 if __name__ == "__main__":
     update_urdf()
+
+    path = Path("sim/stompy_legs/robot.urdf")
```

Comment:
move this to a separate file that will use create_mjcf and create_urdf

Reply:
fixed